### DISCUSSION TITLE
some cool ideas

### DISCUSSION THREAD

**Discussion by mercabun** [keiwando/toonsquid#314] [Ideas]
export as PSD file would be very cool since i mostly storyboard in the app 
Blur tool, the soften within selection tool doesn't actually work (or at least it isn't working for me)
shift-and-trace tool  that makes it easier to track if your animation is not aligning to the movement of your animation (im really just taking this from toonboom but it's very helpful, definitely make something similar to it)

overal great app :0) cant wait to continue using it

**Comment by DinoAwakening** [comment]
> export as PSD file would be very cool since i mostly storyboard in the app Blur tool, the soften within selection tool doesn't actually work (or at least it isn't working for me) shift-and-trace tool that makes it easier to track if your animation is not aligning to the movement of your animation (im really just taking this from toonboom but it's very helpful, definitely make something similar to it)
> 
> overal great app :0) cant wait to continue using it

You just nailed with the “Blur Tool”. Also, we may also need quality of life features such as custom number of joints on “Warp” transformation. Maybe we could get a bucket fill tool rework, so it might be actually reliable. I made a video aswell a long time ago about some features I’d like to see in ToonSquid :D 


https://youtu.be/i2ZlolZOH98

**Reply by 6xolotl** [reply]
Blur tool and custom number of warp joints both sound like lifesavers 🙏
